i18n/controller.go

```diff
@@ -20,6 +24,10 @@ func (controller *i18nController) Update(context *admin.Context) {
 		}
 	}
 
-	controller.I18n.SaveTransaltion(&translation)
+	err := controller.I18n.SaveTranslation(&translation)
+	if err != nil {
```

Comment:
i don't know. is the `context.Writer.Write([]byte("OK"))` at the end interpreted somewhere? or is it just to send something?

do we have something like 

```
qor.Error {
    Status int
    error error
...
}
```
we should probably at least set an `http.Status` other than 200